podcasts/ProfileViewController.swift

```diff
@@ -325,7 +331,7 @@ class ProfileViewController: PCViewController, UITableViewDataSource, UITableVie
 
     private func tableData() -> [[ProfileViewController.TableRow]] {
         var data: [[ProfileViewController.TableRow]]
-        data = [[.allStats, .downloaded, .uploadedFiles, .starred, .listeningHistory]]
+        data = [[.allStats, .downloaded, .uploadedFiles, .starred, .listeningHistory, .help]]
```

Comment:
What do you think about having the help cell in its own section?

```suggestion
        data = [[.allStats, .downloaded, .uploadedFiles, .starred, .listeningHistory], [.help]]
```

https://github.com/Automattic/pocket-casts-ios/assets/7040243/17b9b63d-d4bd-4fef-a9db-d3cd6b4ce477

I had to make a few other changes for this to work, documenting them here:

```swift
    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
        if EndOfYear.isEligible && indexPath.section == 0 && indexPath.row == 0 {
            return UITableView.automaticDimension
        } else {
            return 70
        }
    }

    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
        if section == 0 {
            return headerView
        }

        return nil
    }

    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
        if section == 0 {
            return headerViewModel.contentSize?.height ?? UITableView.automaticDimension
        }

        return UITableView.automaticDimension
    }
```